### ISSUE TITLE
Search Results and Pagination - 4.1.3 Status Message - Screen reader does not read out number of results

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9726]
As a user reliant on a screen reader, I should be able to determine the number of results (be that many or none) displayed when I enter a search term.

**Linked Work Item:**
#9659 

**Expected Result:**
When a user enters a valid or non-valid search term, selects a result from the drop-down and hits Enter, the screen reader should then read out the number of results returned by the user's search.

**Actual Result:**
The number of search results is not read out by the screen reader. Instead, the screen reader reads, "Main landmark, Search results region, edit blank" (or something to that effect) every time, no matter how many results are displayed.

**Evidence:**

(Use NVDA to replicate)

**Comment by apeters** [comment]
closed by #9796

**Comment by imogenroberts** [comment]
RETEST: Screen reader reads out the number of items shown in the dropdown list when a search term is entered, but not how many a results are displayed when the user selects a search term and all records are displayed. This was the fault of poor explanation on my part - a more detailed description of the nature of the defect has been given above.

**Comment by johnatawnclementawn** [comment]
@imogenroberts Is it enough for the screen reader to read out something like `39 total results, 5 shown`?

**Comment by imogenroberts** [comment]
@johnatawnclementawn yes, that would be perfect.

**Comment by johnatawnclementawn** [comment]
Resolution improved by #9810

**Comment by imogenroberts** [comment]
RETEST: Results are read out by the screen reader eventually, but after a long list of repeated components (e.g. letting the user know that they can remove their search term, which part of the page they are in and reading out the name of a combobox). Only after all this is read out are the number of results read out. It is sufficient for only the search term entered and the number of results to be read out (even only the number of results if we are pressed for time). 

https://github.com/archesproject/arches/assets/77161085/a4739717-a086-466d-bd22-153987d18c01

**Comment by apeters** [comment]
@imogenroberts I changed the aria-live property to `assertive` rather than `polite`.  That seems to work better in some instances and should read out the results count sooner.  Can you pull the latest and retest?

**Comment by imogenroberts** [comment]
RETEST: Changing the aria-live to assertive has much improved the interpretation of the results, as the results are read out much sooner.

***Evidence:**


https://github.com/archesproject/arches/assets/77161085/f06e4a94-0d9b-4406-8b5d-3385a6cf1390
